src/components/menus/PlanMenu.svelte

```diff
@@ -53,6 +55,11 @@
     effects.createPlanSnapshot(plan, user);
   }
 
+  async function exportPlan() {
```

Comment:
The three `exportPlan` function definitions vary in behavior – could we standardize this or move it to an effect? The biggest difference is that the one in PlanForm involves showing progress whereas the others do not. We should probably be consistent about displaying or not displaying export progress.

Reply:
If you try a longer running export it feels odd to not have a progress indicator, could see users hitting the export button a few times, waiting a bit, seeing nothing, and then giving up.